src/core/edit/edit.cell.view.js

```diff
@@ -260,6 +264,9 @@ export class EditCellView extends View {
 	}
 
 	contextFactory(cell, value, label, tag) {
+		if (!cell) {
```

Comment:
this is not right context should always return context not boolean, if cell is undefined createContext shouln't be called